engine/runtime-language-epb/src/test/java/org/enso/interpreter/epb/EpbLanguageDependenciesTest.java

```diff
@@ -0,0 +1,49 @@
+import static org.junit.Assert.fail;
+
+import org.junit.Test;
+
+public class EpbLanguageDependenciesTest {
+  public EpbLanguageDependenciesTest() {}
+
+  @Test(expected = ClassNotFoundException.class)
+  public void avoidFlatbuffers() throws Exception {
```

Comment:
There are the dependencies one gets when one `dependsOn(polyglot-api)`. Checking that most of them is now gone.